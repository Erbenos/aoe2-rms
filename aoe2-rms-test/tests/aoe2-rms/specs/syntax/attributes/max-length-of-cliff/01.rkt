#lang aoe2-rms

<CLIFF-GENERATION>
(max-length-of-cliff 3)
