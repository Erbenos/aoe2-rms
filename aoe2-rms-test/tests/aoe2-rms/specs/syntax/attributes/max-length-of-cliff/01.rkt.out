<CLIFF_GENERATION>
max_length_of_cliff 3
